roles/ceph-infra/tasks/configure_firewall.yml

```diff
@@ -179,7 +179,14 @@
     firewalld:
       port: "3000/tcp"
       zone: "{{ ceph_dashboard_firewall_zone }}"
-      source: "{{ public_network }}"
+      permanent: true
+      immediate: true
+      state: enabled
+
+  - name: Open ports for node_exporter
+    firewalld:
+      port: "9100/tcp"
+      zone: "{{ ceph_dashboard_firewall_zone }}"
```

Comment:
The same, here for node_exporter (`when: dashboard_enabled`) and any other dashboard-related ports.